.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,3 +30,9 @@
     - [ ] N/A or can only be tested interactively
```

Comment:
@TylerLeonhardt, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Common Tests - Validate Markdown Files.Should not have errors in any markdown files
```powershell
Expected strings to be the same, but they were different.
Expected length: 9
Actual length:   229
Strings differ at index 0.
Expected: '--EMPTY--'
But was:  '/home/vsts...'
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown.tests.ps1: line 96
96:         $markdownErrors -join "`n" | Should -BeExactly "--EMPTY--"
```

Reply:
@TravisEz13 any idea how I can figure out what it doesn’t like?

Reply:
You have a top level bullet point.  that is not allowed.  Please use the markdownlint extension.

Reply:
I do. It reports nothing.

Reply:
I think I found it…